apis/core.oam.dev/common/types.go

```diff
@@ -98,6 +98,17 @@ type Helm struct {
 	Repository runtime.RawExtension `json:"repository"`
 }
 
+// Terraform is the struct to describe cloud resources managed by Hashicorp Terraform
+type Terraform struct {
+	// Configuration is Terraform Configuration
+	Configuration string `json:"configuration"`
```

Comment:
I'm not sure. Does the terraform user familiar with the name `configuration`? If not,  will `HCL` be a better name?

Reply:
`Configuration` is the standard term. 
>A Terraform configuration is a complete document in the Terraform language that tells Terraform how to manage a given collection of infrastructure. A configuration can consist of multiple files and directories.

https://www.terraform.io/docs/language/index.html#about-the-terraform-language

It consists of `Configuration Syntax` and `JSON Configuration Syntax`. So make it clear, I choose `Configuration`. The `Type` indicates configuration type, `hcl` or `json` and the default is `hcl`.

We can make a survey in China OAM bi-weekly meeting and get some feedback.

Reply:
Cool, then it's fine to use `Configuration `